src/components/paintings-page/painting-tile/painting-tile.module.scss

```diff
@@ -14,7 +14,7 @@
   }
 }
 
-.textContainer {
+.recommendedTextContainer {
```

Comment:
Removing the original classname creates a bug - check out the painting tiles on the paintings page, the text container is missing these styles.